packages/merge-styles/etc/merge-styles.api.md

```diff
@@ -0,0 +1,168 @@
+## API Review File for "@uifabric/merge-styles"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+declare function concatStyleSets<TStyleSet extends IStyleSet<TStyleSet>>(styleSet: TStyleSet | false | null | undefined): IConcatenatedStyleSet<TStyleSet>;

```

Comment:
This is cool!  I think it's an improvement over the .api.ts files (also doesn't make VS Code display tons of errors).